dailizabot/text_patterns.py

```diff
@@ -13,4 +13,20 @@
     "Würde {0} dir denn wirklich helfen?",
     "Bist du sicher, dass du {0} brauchst?"]],
 
+    [r"Du bist (.*)", 
+     ["Wieso denkst du das {0}?",
+      "Kann ich etas dagegen tun {0}?", 
```

Comment:
```suggestion
      "Kann ich etwas gegen {0} tun?", 
```
Spelling mistake & in my opinion the sentence sounds better like that!